lib/structures/Guild.js

```diff
@@ -6,6 +6,8 @@ const GuildChannel = require("./GuildChannel");
 const Collection = require("../util/Collection");
 const Member = require("./Member");
 const Role = require("./Role");
+const ImageFormats = require("../Constants").ImageFormats;
+const ImageSizes = require("../Constants").ImageSizes;
```

Comment:
Might as well import Constants and do `Constants.ImageXYZ` in the rest of the code